eng/pipelines/templates/build-official-release.yml

```diff
@@ -1,60 +1,135 @@
+        publishVstsFeed: DevDiv/vs-green
+        publishFeedCredentials: azure-public/vs-impl
+        allowPackageConflicts: true
+        nuGetFeedType: external
+        
+
   steps:
 
   ###################################################################################################################################################################
   # Validation of insertion branches
   ###################################################################################################################################################################
 
   # See: https://stackoverflow.com/a/30524983/294804
-  - powershell: if (-not (git ls-remote --heads https://$(System.AccessToken)@dev.azure.com/devdiv/DevDiv/_git/VS $(InsertionVSBranch))) { Write-Host "The branch name '$(InsertionVSBranch)' is not valid."; exit 1 }
+  - powershell: if (-not (git ls-remote --heads https://$(System.AccessToken)@dev.azure.com/devdiv/DevDiv/_git/VS ${{ parameters.InsertionVSBranch }})) { Write-Host "The branch name '${{ parameters.InsertionVSBranch }}' is not valid."; exit 1 }
     displayName: Validate VS Insertion Branch Name
-    condition: eq(variables.CreateVSInsertion, true)
-  - powershell: if (-not (git ls-remote --heads https://$(System.AccessToken)@dev.azure.com/devdiv/DevDiv/_git/vs-green $(InsertionVSGreenBranch))) { Write-Host "The branch name '$(InsertionVSGreenBranch)' is not valid."; exit 1 }
+    condition: eq(${{ parameters.CreateVSInsertion }}, true)
+  - powershell: if (-not (git ls-remote --heads https://$(System.AccessToken)@dev.azure.com/devdiv/DevDiv/_git/vs-green ${{ parameters.InsertionVSGreenBranch }})) { Write-Host "The branch name '${{ parameters.InsertionVSGreenBranch }}' is not valid."; exit 1 }
     displayName: Validate vs-green Insertion Branch Name
-    condition: eq(variables.CreateVSGreenInsertion, true)
-
-  ###################################################################################################################################################################
-  # PLUGIN INSTALLATION
-  ###################################################################################################################################################################
-
-  # Use a disabled, empty script to display the section header in the pipeline UI.
+    condition: eq(${{ parameters.CreateVSGreenInsertion }}, true)
+    
+    # Use a disabled, empty script to display the section header in the pipeline UI.
   - script:
     displayName: === Plugin Installation ===
     condition: false
-
-  # Necessary for signing the assemblies and packages.
-  # YAML reference: https://devdiv.visualstudio.com/Engineering/_git/MicroBuild?path=/src/Tasks/SigningPlugin/task.json
+    
+    # Necessary for signing the assemblies and packages.
+    # YAML reference: https://devdiv.visualstudio.com/Engineering/_git/MicroBuild?path=/src/Tasks/SigningPlugin/task.json
   - task: MicroBuildSigningPlugin@4
```

Comment:
PR discussion: the tasks beginning in "MicroBuild" are all done by the 1ES template, so we can remove here :)